### PR TITLE
Rename aspnet5 to aspnet core 1

### PR DESCRIPTION
**Pull request by asierba**
I just updated the title in the docs to the new  "aspnet core 1.0".

I really like the docs in docs.asp.net. Thanks for all your work! :)

### CODE REVIEW

**Comment by zhouyongtao** [comment]
估计得RC2以后还能变名称，现在所有的文档名字还是ASP.NET5 呢，另外DNX也得更换到CLI啊，VS15 是不是也得出一个SP2啥的 ，等中国年以后吧 。

**Comment by danroth27** [comment]
I think in the docs we are going to replace ASP.NET 5 with ASP.NET Core without any version number. The version selector will tell you which version you are reading about based on the doc branches and tags courtesy of [Read the Docs](http://readthedocs.com).

We are going to hold off on updating the public docs with the new branding until we ship the next preview release with the name changes. But we are planning to make these updates in our dev branch.

Could update this PR to be based on dev instead of master and leave off the version number?

File: aspnet/index.rst [hunk 1]
```diff
@@ -5,10 +5,10 @@
 
 .. _index:
 
-ASP.NET 5 Documentation
+ASP.NET Core 1.0 Documentation
 =======================
```
Comment (danroth27) [inline aspnet/index.rst]: The underline need to be the full length of the [title or section header](http://www.sphinx-doc.org/en/stable/rest.html#sections).

**Comment by asierba** [comment]
OK thanks. I'll create a new PR pointing to dev with those changes.

**Comment by asierba** [comment]
This is the new PR
https://github.com/aspnet/Docs/pull/942

I will close this one.

**Pull request closed without merging**